process/working_groups.md

```diff
@@ -6,24 +6,27 @@ parent: Processes
 # ASWF Working Groups
 _Modeled off of CNCF Working Group model ( https://github.com/cncf/toc/blob/master/workinggroups/README.md )_
 
-## Introduction
-
 The purpose of working groups are to study and report on a particular question and make recommendations based on it's findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing it's findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a [TAC member] and with an affirmative vote of the ASWF TAC. The TAC can also shut down a working group with a super majority vote.
 
-## Process
+## Forming a working group
+
+Interested parties in forming a working groups should align with the interested parties on the [charter template]. Once the working group charter is agreed upon, and a [TAC member] is identified as a sponsor, it should be shared with the [TAC email list] to be added to the agenda for the next TAC meeting for review and approval. A working group will not be designated and recognized as an official ASWF Working Group until the TAC approves the charter.
+
+## Working Group activities and collaboration
+
+Working groups generally will meet on a regular basis and operate in an open and transparent way. ASWF provides working groups access to Zoom, a mailing list, Slack channel, wiki space, and GitHub repo. Please submit a [request on the foundation repo](https://github.com/AcademySoftwareFoundation/foundation/issues/new) to have this infrastructure created.
```

Comment:
Is that the best way to go? No issues have been filed so far against that repo, and I'm afraid few people know it exists. Would a LF ticket be a more direct way?

Reply:
I'm trying to drive more folks this way as it's easier to track and manage. The LF ticket ends up being more useful for those cases where other LF departments need to engage.